src/types/index.ts

```diff
@@ -69,7 +69,7 @@ export enum DevicePermissionStatus {
 }
 
 export enum DeviceLabels {
-  None,
+  None = 1,
```

Comment:
Does this need to be documented in migration? Or should be just fix the `if` check?